control/control.yaml

```diff
@@ -95,3 +95,26 @@ components:
             Route specific states
           type: string
           enum: [withdraw, advertise]
+
+    Protocol.State:
+      description: >-
+        Sets the device protocol peer state
+      type: object
+      required: [state]
+      properties:
+        names:
+          description: >-
+            The names of device protocol peer objects to control.
+            If no names are specified then all protocol peer objects that match the
+            x-constraint will be affected.
+          type: array
+          items:
+            type: string
+          x-constraint:
+          - '/components/schemas/Device.Bgpv4/properties/name'
+          - '/components/schemas/Device.Bgpv6/properties/name'
+        state:
+          description: >-
+            Protocol peer specific states
+          type: string
+          enum: [up, down]
```

Comment:
Is this better described as 'Start' and 'Stop'?  'Up' and 'Down' are the results of the protocol state-machine and also depend on the DuT.

Reply:
I agree that Start and Stop probably make more sense.

Reply:
change it to Start and Stop